### DISCUSSION TITLE
Google Searches

### DISCUSSION THREAD

**Discussion by ChristelleWehbe93** [vercel/community#7243] [Help] [answered]
### Summary

I am new to setting up websites etc and I need some information about how I can make my website visible in Google searches. 
I have set up SEO title, description etc in my code correctly. Do I have to get my own domain in order for the website to be visible in searches or can I still be using vercel.app domain? 

According to some researches I made I have to make some DNS configurations but I am not really understanding how or where these configurations should be done. 

Would really appreciate some help. 
Thank you!

### Steps to Reproduce

no steps

**Comment by versecafe** [comment, marked as answer]
@ChristelleWehbe93 having a unique domain does help but you can show up in results on a `*.vercel.app` domain, make sure you have a `sitemap.xml` configured for search engines as well as a `robots.txt` to declare that your site can be indexed otherwise search engine crawlers will skip right over it. The last thing you can do is verify your site with the google search console in order to get metrics from google on the traffic and confirm it has your sitemap and indexed pages.

**Reply by ChristelleWehbe93** [reply]
Hello!
I installed sitemap and simple-robots to my nuxt project, and did even verify my site at Google Search Console. 
But still no results found if I make a search on Google. 

Do you have any guides or examples I can follow? I am using NuxtJS.

**Reply by versecafe** [reply]
Depending on how old your site is you could be in a "sandbox" mostly isolated, google denies they exist but there is some decent evidence it's a thing, I'd recommend checking on Bing and DuckDuckGo to see if you come up in search results, you can also force trigger google to index pages in the search console to get started as well as use the Vercel toolbar OG menu to double check that all SEO relevant things such as title, description, & image are being correctly detected.

**Reply by ChristelleWehbe93** [reply]
I checked Bing and DuckDuckGo and could not find it there as well. 
I have the following tags added to my website title, description, image and url. I have now requested indexing for my page from the Google Search Console, so will see if that helps.

[a comment by an automated account is not shown]
